components/scream/scripts/gather-all-data

```diff
@@ -38,29 +45,42 @@ OR
     > {0} './scripts/test-all-scream $compiler -m $machine' -l -m $machine
 
     \033[1;32m# Do correctness testing for scream-docs micro apps \033[0m
-    > {0} './test-all $compiler $kokkos' -d
+    > {0} './test-all $compiler $kokkos' -o
 
     \033[1;32m# Do a scaling performance test for lin-interp on blake \033[0m
-    > {0} '$scream/scripts/perf-analysis ncol:8000 km1:128 km2:256 minthresh:0.001 repeat:10 --kokkos=$kokkos --test=lin-interp/li_ref --test=lin-interp/li_kokkos --test=lin-interp/li_vect -s ncol:2:128000' -m blake -d
-""".format(os.path.basename(args[0])),
+    > {0} '$scream/scripts/perf-analysis ncol:8000 km1:128 km2:256 minthresh:0.001 repeat:10 --kokkos=$kokkos --test=lin-interp/li_ref --test=lin-interp/li_kokkos --test=lin-interp/li_vect -s ncol:2:128000' -m blake -o
+""".format(pathlib.Path(args[0]).name),
         description=description,
         formatter_class=argparse.ArgumentDefaultsHelpFormatter
     )
 
-    parser.add_argument("run", help="What to run on the machine, cwd will be either scream-perf-$machine/components/scream or scream-docs-perf-$machine/micro-apps depending on if --scream-docs was provided")
+    parser.add_argument("run", help="What to run on the machine, cwd for the command will be root-dir. "
+                        "Supported magic strings: $compiler $kokkos $machine $scream_docs $scream")
 
-    default_commit = get_current_commit(short=True)
-    parser.add_argument("-c", "--commit", default=default_commit, help="Commit to test. Default is current HEAD.")
+    parser.add_argument("-c", "--commit", help="Commit to test. Default is current HEAD.")
 
     parser.add_argument("-m", "--machine", dest="machines", action="append", choices=MACHINE_METADATA.keys(),
                         help="Select which machines to run on, default is all")
 
-    parser.add_argument("-d", "--scream-docs", action="store_true", help="Test scream-docs instead of scream.")
+    parser.add_argument("-o", "--scream-docs", action="store_true", help="Test scream-docs instead of scream.")
 
     parser.add_argument("-l", "--local", action="store_true",
-                        help="Run tests on local machine. Assumes ./scream and ./scream-docs")
+                        help="Run tests on local machine, only using this script to manage env and batch submission.")
+
+    parser.add_argument("-k", "--kokkos", help="Use to select specific kokkos installation. "
+                        "Default will be scream-repo/externals/kokkos. "
+                        "Supported magic strings: $machine $compiler")
```

Comment:
Does this mean I can  do, say, `-k $HOME/kokkos/$machine/$compiler`?

Reply:
Yep! You'll need to use quotes to ensure the shell doesn't try to expand those variables:
```
-k '$HOME/kokkos/$machine/$compiler'
```